### PR TITLE
Add react toastify

### PR DESCRIPTION
**Pull request by PierreGoe**

### CODE REVIEW

**Review by LuigiMenez — DISMISSED** [review]
Ok for me. Styling the error message, with Toastify React module.

File: src/App.jsx [hunk 1]
```diff
@@ -53,6 +55,19 @@ function App() {
           <FavoritesView favorites={favorites} setFavorites={setFavorites} />
         </Route>
       </Switch>
+      <ToastContainer
+        // theme="dark"
```
Comment (Hacene010) [inline src/App.jsx]: Dead code

**Review by Hacene010 — CHANGES_REQUESTED** [review]
You have a dead code

**Review by LuigiMenez — APPROVED** [review]
Good

**Review by Hacene010 — APPROVED** [review]
Seems ok

**Pull request merged**